client/src/legacy/HtmlEditorField.js

```diff
@@ -92,13 +93,21 @@ ss.editorWrappers.tinyMCE = (function() {
       // Add instance specific data to config
       config.selector = selector;
 
+      const change = debounce(function () {
+        var instance = self.getInstance();
+        jQuery(instance.getElement()).trigger("change");
+        instance.getElement().dispatchEvent(new Event('input', { bubbles: true }));
+      }, 250, { leading: true, trailing: true });
```

Comment:
This isn't strictly necessary - but given the WYSIWYG is where most key events will be happening it seems sensible to debounce this to offset the fact that the debounce is shorter in the changetracker.